content/en/docs/tasks/debug/debug-cluster/_index.md

```diff
@@ -313,7 +327,10 @@ This is an incomplete list of things that could go wrong, and how to adjust your
 - Action: applications (containers) designed to tolerate unexpected restarts
   - Mitigates: Node shutdown
   - Mitigates: Kubelet software fault
+  
+### Things to note after dockershim migration
 
+*Note: If you get the following error after upgrading to Kubernetes 1.24, go through the [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/) guide to get a better understanding of how to resolve this error.*
```

Comment:
Use `{{< note >}}` shortcode instead.

Reply:
Do you mean I need to put the text inside `< >` ?

Reply:
Read https://kubernetes.io/docs/contribute/style/hugo-shortcodes/ on how to use hugo shortcodes